pkg/providers/imagefamily/nodeimage_test.go

```diff
@@ -67,10 +67,14 @@ func getExpectedTestCIGImages(imageFamily string, version string, kubernetesVers
 	return nodeImages
 }
 
+// assuming FIPSMode == Disabled
+//
 //nolint:unparam // might always be using the same version in test, but could change in the future
 func getExpectedTestSIGImages(imageFamily string, version string, kubernetesVersion string) []imagefamily.NodeImage {
```

Comment:
Based on naming, I think `getExpectedTestFIPSImages` should actually be consolidated into `getExpectedTestSIGImages`, and take `fipsMode` as a parameter.

This is because fips images are a subcategory of SIG, so really it would make sense in my opinion to call `getExpectedTestSIGImages` and have it internally determine the difference between the fips case and not.